website/docs/GettingStarted.md

```diff
@@ -40,27 +40,10 @@ In order to properly use helpers for async tests (`findBy` queries and `waitFor`
 
 ### Additional Jest matchers
 
-In order to use additional React Native-specific Jest matchers from [@testing-library/jest-native](https://github.com/testing-library/jest-native) package add it to your project:
+In order to use additional React Native-specific Jest matchers add the following line to your `jest-setup.ts` file:
```

Comment:
It needs to be imported in a setup after env file, I think it may not be very clear here, we could say add the following line to your `jest-setup-after-env.ts`?

Reply:
Good point, I think usually projects should do setup in setupFilesAfterEnv, so I wil clarify this.